fdbserver/Ratekeeper.actor.cpp

```diff
@@ -738,6 +738,7 @@ ACTOR Future<Void> monitorServerListChange(
 	loop {
 		try {
 			tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+			TraceEvent("RatekeeperMonitorSSList", self->id).detail("CurrentTime", now());
```

Comment:
I think it's worth discussing whether this is a good idea:

1. Monitoring process busyness by timing delays is not scalable. There are a ton of delays where we may want to check this, and it doesn't make sense to put trace events on all of them.
2. We should be able to answer the question of whether a process is too busy to answer requests without trace events. As I mentioned in my other comment, we already have metrics for this, but if those are insufficient for our needs we should be improving those generic metrics.
3. This check isn't going to be able to distinguish between delays that happen in the various waits in this function.

Reply:
logging trace event in each loop will not scale or work as @sfc-gh-ajbeamon  mentioned. 

How about this:
At the beginning of each loop, 
* latency = now() - start_time_of_last_loop;
* If latency is larger than 2* SERVER_LIST_DELAY, output a SevWarn event.

The warning shouldn't happen in a healthy cluster. 

The warning can be super helpful to SRE when RK limits the traffic due to its own slowness.

Reply:
Done.